contributors/devel/api_changes.md

```diff
@@ -798,9 +800,12 @@ A developer is considering adding a new `Width` parameter, like this:
 ```go
 // API v6.
 type Frobber struct {
-  Height int    `json:"height"`
-  Width  int    `json:"height"`
-  Param  string `json:"param"`
+  // height ...
+  Height *int32 `json:"height" protobuf:"varint,1,opt,name=height"`
+  // param ...
+  Param  string `json:"param" protobuf:"bytes,2,opt,name=param"`
+  // width ...
+  Width  *int32 `json:"width,omitempty" protobuf:"varint,3,opt,name=width"`
```

Comment:
Do you prescribe adding the new field to the end of the Go struct, with the intention of allowing existing "unkeyed" [composite literals](https://golang.org/ref/spec#Composite_literals) to remain valid when recompiled, or do you mandate use keyed elements so that existing code can't impose assumptions about field order?